backends/build_system/validate_env.py

```diff
@@ -30,12 +31,16 @@
 
 class UnmetDependenciesException(Exception):
     def __init__(self, unmet_deps):
+        pip_install_command = f"{sys.executable} -m pip install "
         msg = "Environment requires following Python dependencies:\n\n"
         for package, actual_version, required in unmet_deps:
             msg += (
                 f"{package} (required: {required.constraints}) "
                 f"(version installed: {actual_version})\n"
             )
+            pip_install_command += f'"{package}{required.string_constraints()}" '
+        msg += "\nRun the following pip command to install manually:\n"
+        msg += f"{pip_install_command}\n"
```

Comment:
Could we also add a test or expand on an existing test case to make sure this pip command is what we expect?